frontend/src/Judge/components/Shows.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import ShowCard from '../components/ShowCard'
+
+const Shows = ({ user, loading }) => (
+  <div>
+    {loading ? null :
+    user.shows.length > 0 ? user.shows.map(show => <ShowCard key={show.id} {...show} />) :
+    "You are not currently assinged to a show"
```

Comment:
I would center this instead of having it off to the side to stay with the flow when cards do show up

Reply:
I'd also center the text (horizontally and vertically). Also, `user.shows.length ? ...` should be fine (0 is falsey)